src/renderer/utils/order-by-priority.ts

```diff
@@ -0,0 +1,17 @@
+/**
```

Comment:
Maybe this file should be located in `common/utils`?

Reply:
Hm, maybe. It's used only in renderer though and that's why it's there.

Reply:
I think code that can be used in both should be placed in common.